resources/lang/fr/Ticket.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+return array (
+  //============================== New strings to translate ==============================//
+  // Defined in file C:\\wamp\\www\\attendize\\resources\\views\\ManageEvent\\Tickets.blade.php
+  'on_sale' => 'On Sale',
+  //==================================== Translations ====================================//
+  'attendee_ref' => 'Réf. participant',
+  'coupon_codes' => 'Codes Coupon',
+  'create_ticket' => 'Créer un billet',
+  'demo_attendee_ref' => '#YLY9U73-1',
+  'demo_end_date_time' => 'Mar 18th 5:08PM',
+  'demo_event' => 'Événement de démo',
+  'demo_name' => 'Le Blog de Bob',
+  'demo_order_ref' => '#YLY9U73',
+  'demo_organiser' => 'Organisateur de démo',
+  'demo_price' => '€XX.XX',
+  'demo_start_date_time' => 'Mar 18th 4:08PM',
```

Comment:
?

Reply:
@jeremyquinton Same question here: how to localize `'Mar 18th 4:08PM'`?